pages/wiki/building-asteroidos.hbs

```diff
@@ -10,35 +10,91 @@ layout: documentation
+
+<p>Build a Docker image called <em>asteroidos-toolchain</em> from the given Dockerfile:</p>
+<pre><code>sudo docker build --tag asteroidos-toolchain .</code></pre>
+
+<p>Remove the Docker container called <em>asteroidos-toolchain</em> in case it already exists:</p>
+<pre><code>sudo docker rm -f asteroidos-toolchain</code></pre>
+
+<div class="page-header">
+  <h2 id="docker-build-usage">Usage example</h2>
+</div>
+<p>Create a Docker container called <em>asteroidos-toolchain</em> from the Docker image which we also called <em>asteroidos-toolchain</em> and build AsteroidOS for <em>dory</em> (the LG G Watch) within the container:</p>
+<pre><code>sudo docker rm -f asteroidos-toolchain ; sudo docker run --name asteroidos-toolchain -it -v /etc/passwd:/etc/passwd -u `id -u`:`id -g` -v "$HOME/.gitconfig:/$HOME/.gitconfig" -v "$(pwd):/asteroid" asteroidos-toolchain bash -c "source ./prepare-build.sh dory && bitbake asteroid-image"</code></pre>
+
+<p>Note: The files created during the build simply go to the current direcotry (which should be <em>asteroid</em>), more spcifically, the output goes to the subdirectories <em>asteroid/src</em> and <em>asteroid/build</em>.</p>
+<p>To build for a different watch than the LG G Watch, use its corresponding codename instead of <em>dory</em> when executing the <em>docker run ...</em> command.</p>
+<p>You can find the codenames for the supported watched on the <a href="{{rel 'install'}}">Install page</a>.</p>
+
+
+<div class="page-header">
+  <h3 id="docker-run-explained">What exactly does that long <em>docker rm ... ; docker run ...</em> command do?</h3>
+</div>
+<pre><code># sudo docker rm -f asteroidos-toolchain  Removes the Docker container called "asteroidos-toolchain" in case it already exists
+# -it                                     Attaches the terminal to the container so that we can see the output. Otherwise it would run blindly in the background.
+# -u `id -u`:`id -g`                      Ensures that the current user id and group id from the host is used on files inside the container to avoid permission issues.
+# -v /etc/passwd:/etc/passwd              Ensures that the user ids and groups from the host are also available in the Docker container. (Otherwise the -u and -g would be useless.)
+# /etc/passwd                             Contains the user names and their ids.
+# "$HOME/.gitconfig:/$HOME/.gitconfig"    Shares your user's git config with the container.
+# -v "$(pwd):/asteroid"                   Is used to mount the current directory (which is your asteroid git repo clone) into the container.
+
+# bash -c "source ./prepare-build.sh dory && bitbake asteroid-image"   This is the command to be executed inside of the container.
+                                                                       "dory" being the codename of your watch.
+                                                                       (You have to use source and not just "." here!)</code></pre>
```

Comment:
I don't think it's necessary to tell users not to run a command that is not even given

Reply:
If you don't like it, I'll remove. But I thought it would be a good idea to mention it because in other places of the documentation `. ./prepare-build.sh` is used and it wasn't obvious to me that the `.` doesn't behave the same as `source` within a `bash -c "..."` call. For instance I could see please copying `. ./prepare-build.sh dory` and `bitbake meta-toolchain-qt5` from the `Creating an Asteroid App`-page.

Reply:
Let's just replace . with source in the couple of remaining places. Half of the users miss the dot and complain that it doesn't work anyway.